x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/package_policy_input_panel.tsx

```diff
@@ -104,6 +104,7 @@ export const PackagePolicyInputPanel: React.FunctionComponent<{
       [packageInputStreams, packagePolicyInput.streams]
     );
 
+    console.log(packagePolicyInput);
```

Comment:
probably want to remove that one

Reply:
I don't know how that passed the commit precheck